datasets/gov/giscentras/grpk.csv

```diff
@@ -0,0 +1,346 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/giscentras/grpk,,,,,,,,,,,,,
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,vadovas,,,,,https://atviriduomenys.readthedocs.io/,,
+,,,,,,,,,,,,,,
+,,,,Kelias,,,,,,,,,KELIAI,Sluoksnyje saugoma informacija apie kelių ir gatvių ašines linijas
+,,,,,,comment,type,,"create(resource: ""file"", type: ""shape"", ref: ""lapai"", source: ""{}.shp"")",,open,spinta:283,,

```

Comment:
Neaišku ką reiškia šis komentaras? Ar tai reiškia, kad duomenys jau atverti ir kažkur publikuojame Shape formatu?

Reply:
Duomenys atverti ir publikuojami shape formatu: https://www.geoportal.lt/download/opendata/GRPK/GRPK_Open_Data.zip